bundles/SimpleBackendSearchBundle/public/js/pimcore/element/service.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+/**
+ * @private
+ */
+pimcore.registerNS('pimcore.bundle.search.element.service');
+
+pimcore.bundle.search.element.service = Class.create({
```

Comment:
combine `pimcore.bundle.search.element.service` and `pimcore.bundle.search.layout.quickSearch` to one service

Reply:
done 9d4d7d1f1318d3f930c83f785b5bba54bceba211